thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -76,6 +76,40 @@ struct DeviceGroupHashInfo {
     2: required string hashClass
 }
 
+struct FetchDataPartitionReq {
+    1: required map<string, list<i64>> devicePathsToStartTimeMap
+}
+
+struct FetchSchemaPartitionReq {
+    1: required list<string> devicePaths
+}
+
+struct FetchPartitionReq {
+    1: required map<string, list<i64>> devicePathsToStartTimeMap
+}
+
+struct RegionInfo {
+    1: required i32 regionId
+    2: required list<rpc.EndPoint> endPointList
+}
+
+struct DataPartitionInfoResp {
+    // Map<StorageGroup, Map<DeviceGroupID, Map<TimePartitionId, List<DataRegionReplicaInfo>>>>
+    1: required map<string, map<i32, map<i64, list<RegionInfo>>>> dataPartitionMap
+}
+
+struct SchemaPartitionInfoResp {
+    // Map<StorageGroup, Map<DeviceGroupID, SchemaRegionPlaceInfo>>
+    1: required map<string, map<i32, RegionInfo>> schemaPartitionInfo
+}
```

Comment:
The deviceId of i32 can't support wildcard match of ParthPattern. Maybe should replace it with the raw devicePath

Reply:
If you have `*` in devicePath, you will need to send your request to all the RegionInfo.